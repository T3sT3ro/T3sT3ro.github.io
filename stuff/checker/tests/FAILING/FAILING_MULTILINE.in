Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec interdum elit at tristique lacinia. Sed auctor sem non eros facilisis, id sollicitudin nunc malesuada. Praesent ut vestibulum diam, et euismod eros. Morbi id dui ut ipsum porta volutpat. Aliquam elementum sagittis velit, non sagittis nisi congue eu. Nulla pharetra velit eget arcu interdum volutpat. Ut sit amet velit non nulla finibus volutpat. Suspendisse potenti. Donec cursus odio vitae felis tempus dictum. Morbi vel lacinia velit. Sed et nisi aliquam, egestas augue nec, semper ex. Ut condimentum urna nec quam dictum, ac accumsan metus rhoncus. Integer luctus nisl eros. Praesent sodales eu turpis eget vehicula. Donec sollicitudin vehicula sapien eget semper.

Ut vel pharetra lectus. In dapibus, ipsum condimentum pellentesque rhoncus, orci quam semper dui, ac dapibus tellus sapien sed eros. Duis eu metus nec felis convallis venenatis sed sit amet nunc. Mauris ultrices nisl at erat volutpat elementum. Proin tempus justo ut dolor varius blandit vel ut sem. Curabitur blandit gravida elementum. Duis libero neque, tincidunt viverra neque quis, congue gravida lorem. Proin imperdiet purus venenatis nulla elementum faucibus id quis urna. Etiam erat magna, ultrices ut dignissim id, scelerisque quis magna. Morbi quis urna sit amet lorem suscipit aliquam. Aenean rhoncus quam id tellus volutpat, vitae imperdiet risus sagittis. Sed lobortis non est a placerat.

Pellentesque viverra, sem eu sagittis tempor, nulla erat dapibus risus, at vehicula est augue non arcu. Nam non lorem eget purus pellentesque auctor. Maecenas non quam id ex interdum feugiat in auctor eros. Nullam et enim at urna ullamcorper tincidunt. Maecenas aliquet porta venenatis. Interdum et malesuada fames ac ante ipsum primis in faucibus. Sed id orci ut ligula consectetur rhoncus. Nam porta, metus in facilisis dapibus, purus nibh egestas libero, eget consequat magna dolor nec sapien. Vestibulum vel justo eleifend, mollis augue ut, congue massa. Duis consectetur magna magna, tempor finibus tortor varius quis. Curabitur auctor vitae nisl nec sollicitudin. Aenean finibus nibh libero, nec ultricies felis tempus non. Nulla vitae orci in nisl tempus bibendum vitae vitae nisl. In aliquam interdum libero, ut elementum velit vehicula eu.