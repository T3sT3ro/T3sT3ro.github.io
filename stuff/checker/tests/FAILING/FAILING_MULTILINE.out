
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec interdum elit at tristique lacinia. Sed auctor sem non eros facilisis, id sollicitudin nunc malesuada. Praesent ut vestibulum diam, et euismod eros. Morbi id dui ut ipsum porta volutpat. Aliquam elementum sagittis velit, non sagittis nisi congue eu. Nulla pharetra velit eget arcu interdum volutpat. Ut sit amet velit non nulla finibus volutpat. Suspendisse potenti. Donec cursus odio vitae felis tempus dictum. Morbi vel lacinia velit. Sed et nisi aliquam, egestas augue nec, semper ex. Ut condimentum urna nec quam dictum, ac accumsan metus rhoncus. Integer luctus nisl eros. Praesent sodales eu turpis eget vehicula. Donec sollicitudin vehicula sapien eget semper.

Ut vel pharetra lectus. In dapibus, ipsum condimentum pellentesque rhoncus, orci quam semper dui, ac dapibus tellus sapien sed eros. Duis eu metus nec felis convallis venenatis sed sit amet nunc. Mauris ultrices nisl at erat volutpat elementum. Proin tempus justo ut dolor varius blandit vel ut sem. Curabitur blandit gravida elementum. Duis libero neque, tincidunt viverra neque quis, congue gravida lorem. Proin imperdiet purus venenatis nulla elementum faucibus id quis urna. Etiam erat magna, ultrices ut dignissim id, scelerisque quis magna. Morbi quis urna sit amet lorem suscipit aliquam. Aenean rhoncus quam id tellus volutpat, vitae imperdiet risus sagittis. Sed lobortis non est a placerat.

Pellentesque viverra, sem eu sagittis tempor, nulla erat dapibus risus, at vehicula est augue non arcu. Nam non lorem eget purus pellentesque auctor. Maecenas non quam id ex interdum feugiat in auctor eros. Nullam et enim at urna ullamcorper tincidunt. Maecenas aliquet porta venenatis. Interdum et malesuada fames ac ante ipsum primis in faucibus. Sed id orci ut ligula consectetur rhoncus. Nam porta, metus in facilisis dapibus, purus nibh egestas libero, eget consequat magna dolor nec sapien. Vestibulum vel justo eleifend, mollis augue ut, congue massa. Duis consectetur magna magna, tempor finibus tortor varius quis. Curabitur auctor vitae nisl nec sollicitudin. Aenean finibus nibh libero, nec ultricies felis tempus non. Nulla vitae orci in nisl tempus bibendum vitae vitae nisl. In aliquam interdum libero, ut elementum velit vehicula eu.

In lobortis aliquet lacus, in tristique lacus. Pellentesque nec eleifend ante. Quisque varius blandit neque sit amet porta. Vestibulum quis turpis venenatis neque lacinia dapibus. In vel ex ornare, gravida leo ac, tincidunt ante. Pellentesque malesuada ipsum faucibus justo maximus, facilisis volutpat quam porttitor. Aliquam erat volutpat.

Phasellus imperdiet, lacus eu molestie faucibus, neque lacus tempus magna, sed varius augue lacus eu odio. Phasellus eget elementum metus, nec lacinia velit. Pellentesque consequat ultrices imperdiet. Fusce ipsum nisi, feugiat rhoncus accumsan eget, dictum commodo nunc. Pellentesque imperdiet luctus finibus. Morbi nec neque sed est pellentesque malesuada sit amet id dolor. Duis id lorem non neque gravida condimentum. In sodales sem a augue blandit, id dignissim tortor fringilla.